python/mxnet/rnn/rnn_cell.py

```diff
@@ -1076,3 +1154,143 @@ def __call__(self, inputs, states):
             state to next step of RNN.
         """
         raise NotImplementedError
+
+class BidirectionalCell(BaseRNNCell):
+    """Bidirectional RNN cell
+
+    Parameters
+    ----------
+    l_cell : BaseRNNCell
+        cell for forward unrolling
+    r_cell : BaseRNNCell
+        cell for backward unrolling
+    prefix : str, default 'bi_'
+        prefix for name of node
+        (and prefix of output)
+    """
+    def __init__(self, l_cell, r_cell, params=None, output_prefix='bi_'):
+        super(BidirectionalCell, self).__init__('', params=params)
+        self._override_cell_params = params is not None
```

Comment:
I thought this was for feeding in params of both cells from bidirectional cell. What does the SequentialRNNCell's params field do then?